certified-connectors/WithoutWire Inventory Platform/apiProperties.json

```diff
@@ -0,0 +1,44 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "baseurl": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Base Url",
+          "description": "Base Url (https://api.withoutwire.com/)",
+          "tooltip": "Provide bas url",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
@PavloNikulinEnavate Logo must contain a non-transparent, non-white background color (#ffffff) that matches your specified icon background color.